Gems/WhiteBox/Code/Source/EditorWhiteBoxComponentMode.cpp

```diff
@@ -16,18 +16,27 @@
 #include <AzCore/Settings/SettingsRegistry.h>
 #include <AzCore/std/sort.h>
 
+#include <AzToolsFramework/ActionManager/Action/ActionManagerInterface.h>
+#include <AzToolsFramework/ActionManager/Menu/MenuManagerInterface.h>
+#include <AzToolsFramework/ActionManager/HotKey/HotKeyManagerInterface.h>
```

Comment:
Should we consider a single include that pulls these? `#include <AzToolsFramework/ActionManager/ActionManager.h>`